### ISSUE TITLE
Unable to locate library 'dongle'

### ISSUE THREAD

**Issue by ober** [dbmcclain/AONT-Tolstoy#1]
Where could one find that?

**Comment by dbmcclain** [comment]
Hi Jaime,

Not sure why you would need it, but here it is…

**Comment by ober** [comment]
If it's not needed, can I remove the reference to it in the 
deliver-aont.lisp?
On Sat, 12 Mar 2016, dbmcclain wrote:

> Date: Sat, 12 Mar 2016 22:31:53 -0800
> From: dbmcclain notifications@github.com
> Reply-To: dbmcclain/AONT-Tolstoy
>     <reply+000297a2fa3e416852345dc43539c9f82f124cea4734ef9492cf0000000112fccb5
>     992a169ce085e86a8@reply.github.com>
> To: dbmcclain/AONT-Tolstoy AONT-Tolstoy@noreply.github.com
> Cc: Jaime Fournier jaimef@linbsd.org
> Subject: Re: [AONT-Tolstoy] Unable to locate library 'dongle' (#1)
> 
> Hi Jaime,
> 
> Not sure why you would need it, but here it is…
> 
> > On Mar 12, 2016, at 13:02, Jaime Fournier notifications@github.com wrote:
> > 
> > Where could one find that?
> > 
> > —
> > Reply to this email directly or view it on GitHub https://github.com/dbmcclain/AONT-Tolstoy/issues/1.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/dbmcclain/AONT-Tolstoy/issues/1#issuecomment-195892321

Jaime Fournier
415-889-3948

**Comment by dbmcclain** [comment]
Yes, I’m sure you could. The “Dongle” was from years ago and was needed to compile my Butterfly and Okeanos projects fully. Without a dongle, it would produce a restricted version. And since I always load up Butterfly and Okeanos, I just stuck that dongle into my init file. That’s probably why you ran into it. Just laziness and sloppiness on my part.
- DM
